samples/bluetooth/broadcast_audio_source/Kconfig

```diff
@@ -28,4 +28,13 @@ config ENABLE_LC3
 	select LIBLC3
 	select FPU
 
+config USE_USB_AUDIO_INPUT
+	bool "Use USB Audio as input"
+	# By default, use the USB Audio path is disabled.
+	default n
+	depends on ENABLE_LC3
```

Comment:
Rather than doing 
```c
CONFIG_USE_USB_AUDIO_INPUT=y
```
 in `samples/bluetooth/broadcast_audio_source/boards/nrf5340_audio_dk_nrf5340_cpuapp.conf`, you could follow the approach above and do

```
default y if SOC_NRF5340_CPUAPP
```

So that is is applied for both the regular and audio kit